src/app/SharpRaven/Data/IJsonPacketFactory.cs

```diff
@@ -93,10 +93,11 @@ public interface IJsonPacketFactory
         /// </summary>
         /// <param name="project">The project.</param>
         /// <param name="event">The event to capture.</param>
+        /// <param name="breadcrumbsRecords">Trail to more investigation.</param>
         /// <returns>
         /// A new instance of <see cref="JsonPacket" /> for the specified
         /// <paramref name="project" />, with the given <paramref name="event" />.
         /// </returns>
-        JsonPacket Create(string project, SentryEvent @event);
+        JsonPacket Create(string project, SentryEvent @event, List<BreadcrumbsRecord> breadcrumbsRecords);
```

Comment:
I'd like to not add more parameters to the public interface. Can you move this to the `SentryEvent` class instead?